speech-assets/UtterancesTemplate.txt

```diff
@@ -203,6 +203,21 @@ AggregationIntent {Aggregate} (entries in|rows in) {TableName} \
   {ComparisonColumnThree} {ComparatorThree} [{ColumnValueThree}|{ColumnNumberThree}] \
   {GroupBy} {GroupByColumn}
 
+AggregationIntent {Aggregate} the {AggregationColumn} (column) in {TableName} \
+  (table) \
+  where {ComparisonColumnOne} {ComparatorOne} [{ColumnValueOne}|{ColumnNumberOne}] \
+  {BinaryLogicOpOne} \
+  {ComparisonColumnTwo} {ComparatorTwo} [{ColumnValueTwo}|{ColumnNumberTwo}] \
+  {BinaryLogicOpTwo} \
+  {ComparisonColumnThree} {ComparatorThree} [{ColumnValueThree}|{ColumnNumberThree}] \
+  {GroupBy} {GroupByColumn}
+
+NarrowIntent [What if|What about|How about|And if|And when|Also if|Also when|Also \
```

Comment:
i think lets not support it for now, the order of operations for it is a bit weird / confusing for the user